skimage/segmentation/random_walker_segmentation.py

```diff
@@ -313,13 +313,12 @@ def random_walker(
 
     Parameters
     ----------
-    data : array_like
-        Image to be segmented in phases. Gray-level `data` can be two- or
-        three-dimensional; multichannel data can be three- or four-
-        dimensional with `channel_axis` specifying the dimension containing
+    data : (M, N[, P][, C]) ndarray
+        Image to be segmented in phases. Gray-level `data` can be 2-/3-D; multichannel
```

Comment:
I would have left the long-form descriptions of dimensionality... It didn't hurt.